Bottom: 32322089b569cc61f22b2e08ffd7998c19e29e6e
Top:    289d8bd380a87990cd1cc50667e6d395495c6d7e
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-24 13:58:41 +0800

Refresh of autofs-5.0.7-remove-hard-coded-CFLAGS-and-LDFLAGS.patch

---

diff --git a/Makefile.rules b/Makefile.rules
index 9aebd19..d00ba3c 100644
--- a/Makefile.rules
+++ b/Makefile.rules
@@ -25,16 +25,16 @@ endif
 ifdef DEBUG
 CFLAGS    ?= -g -Wall -DDEBUG
 LDFLAGS   ?= -g
-STRIP     = :
+STRIP     ?= :
 else
 ifdef DONTSTRIP
 CFLAGS    ?= -O2 -g
 LDFLAGS   ?= -g
-STRIP     = :
+STRIP     ?= :
 else
 CFLAGS    ?= -O2 -Wall
 LDFLAGS   ?= -s
-STRIP     = strip --strip-debug
+STRIP     ?= strip --strip-debug
 endif
 endif
